plugins/azure-devops/src/hooks/useRepoBuilds.ts

```diff
@@ -26,6 +26,9 @@ import { useApi } from '@backstage/core-plugin-api';
 import useAsync from 'react-use/lib/useAsync';
 import { getAnnotationValuesFromEntity } from '../utils';
 
+/**
+ * @deprecated Use `useBuildRuns` instead
```

Comment:
As I worked on this I discovered that this hook isn't being used. Just deprecated it for now but should this be in the changeset?

Reply:
It's not being exported (doesn't seem to appear in the api-report file) so it's ok to just outright delete.

Reply:
:face_palm: I've deleted it